paddle/fluid/operators/math/selected_rows_functor.cc

```diff
@@ -412,7 +410,7 @@ struct MergeAdd<platform::CPUDeviceContext, T> {
       out.set_rows(merge_rows);
 
       math::SetConstant<platform::CPUDeviceContext, T> constant_functor;
-      constant_functor(context, out.mutable_value(), 0.0);
+      constant_functor(context, out.mutable_value(), static_cast<T>(0.0));
```

Comment:
There won't be much difference, because this class template is instantiated both for `float` and `double` so either way in one case there would be implicit cast.

Reply:
I was just wondering that with double the converting constructor (bfloat.h:91) would be called, which for double is static_cast to float at first then float one called. Maybe with explicit float we could skip intermediate conversion step, one static_cast and function call, just for performance . What's you opinion?

Reply:
For bfloat16 you're probably right with respect to less conversions. Although for overall performance this would rather won't make much difference. But thanks for pointing out!